library/src/main/kotlin/com/liferay/vulcan/consumer/extensions/Date+.kt

```diff
@@ -0,0 +1,14 @@
+package com.liferay.vulcan.consumer.extensions
+
+import java.text.DateFormat
+import java.util.*
+
+fun Date.shortFormat() = format(DateFormat.getDateInstance(DateFormat.SHORT, Locale.US))
```

Comment:
mmmm we're hardcoding the locale and that's a bad idea. I wouldn't use this extension methods